products/ubuntu2404/product.yml

```diff
@@ -18,7 +18,7 @@ pkg_manager: "apt_get"
 pkg_manager_config_file: "/etc/apt/apt.conf"
 
 init_system: "systemd"
-oval_feed_url: "https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.jammy.cve.oval.xml"
+oval_feed_url: "https://security-metadata.canonical.com/oval/com.ubuntu.noble.cve.oval.xml.bz2"
```

Comment:
Hey @Mab879 The link doesn't match the one in the product stability file, I suspect the latter should be updated too.